.rubocop.yml

```diff
@@ -37,3 +37,6 @@ Style/TrailingCommaInArrayLiteral:
 Rails/CreateTableWithTimestamps:
   Exclude:
     - db/schema.rb
+
+Capybara/ClickLinkOrButtonStyle:
+  EnforcedStyle: link_or_button
```

Comment:
There are accessibility arguments for making sure elements that navigate pages without mutating data use links, and elements that mutate data (either with or without navigation) are buttons.

Unfortunately, I didn't have the brain-power to go through and figure out which of those we want to be what; so I am keeping the current `link_or_button` form that we have been using; rather than the new standard.